.github/workflows/ci.yml

```diff
@@ -2,12 +2,14 @@ name: Continuous integration
 on:
```

Comment:
Can you please undo this file, otherwise GitHub actions does not pick up this PR